Jenkinsfile

```diff
@@ -0,0 +1,129 @@
+@Library('github.com/glogiotatidis/jenkins-pipeline@more-commands')
```

Comment:
As discussed, once https://github.com/mozmar/jenkins-pipeline/pull/2 is finished and merged please s/glogiotatidis/mozmar/ and then update the perms on the jenkins job, and then we should be good to merge this PR.